pennylane/logging/configuration.py

```diff
@@ -99,3 +113,42 @@ def config_path():
     """
     path = os.path.join(_path, "log_config.toml")
     return path
+
+
+def show_system_config():
+    """
+    This function opens the logging configuration file in the system-default browser.
+    """
+    # pylint:disable = import-outside-toplevel
+    import webbrowser
+
+    webbrowser.open(config_path())
+
+
+def edit_system_config(wait_on_close=False):
+    """
+    This function opens the log configuration file using OS-specific editors.
+
+    Setting the `EDITOR` environment variable will override xdg-open/open on
+    Linux and MacOS, and allows use of `wait_on_close` for editor close before
+    continuing execution.
+
+    .. warning::
+
+        As each OS configuration differs user-to-user, you may wish to
+        instead open this file manually with the `config_path()` provided path.
+    """
+    if editor := os.getenv("EDITOR"):
```

Comment:
Is this environment variable normally set up on systems without users having to do anything? It didn't work for me.

Reply:
Most linux systems set this already, or most dev users set this themselves. If not set, the backup is to open the file with the browser.

Reply:
The only data point I have is that it didn't work for me :laughing:

So are we sure this will work out of the box for most users? If not, is this function enough of a utility to be worth it?

Reply:
Seems like parenthesis were needed to properly scope the if-else and walrus operator. Should be fixed now.

Reply:
Can confirm, it works for me now :star_struck: